### ISSUE TITLE
Orifice class

### ISSUE THREAD

**Issue by aldergames** [AlderArts/foe#352]
Combine functions from Vagina and Butt (make them subclasses). Possibly use for Mouth too (make new class). Review and revamp capacity, wetness and stretch functions here. Possibly adjectives too.

Relates to #333

**Comment by aldergames** [comment]
Stretch moved to Orifice class.

**Comment by ElAcechador** [comment]
Can't seem to find any stat relating to wetness at a glance, is there one yet? Also, if we're going to rework capacity and stretching, it might be wise to acknowledge penis length as well, as the current implementation only accounts for girth.

**Comment by aldergames** [comment]
I've intentionally removed wetness, as there isn't really any scene that makes use of it atm, nor do I see how it would give any form of impact other than the throwaway word here and there.

Similar thing for length. Not counting hyper stuff, the main thing restricting penetration is going to be girth. Even if your cock is very long, it's still going to go in - just not all the way. There are some scenes (mainly Kiai) that does take this into account, but most don't bother.

I've actually done the mechanical parts of this rework already. The issue is still up because there's still a bunch of old things floating around that really should be looked over before this is considered "done".

**Comment by ElAcechador** [comment]
Makes sense, even though wetness could have been used to trigger foreplay scenes based on some kind of "wetness requirement" derived from stretching limitations and be skipped otherwise, but then some people might consider this a loss, so I can see how that would have been superfluous.

Can you elaborate some more on the kind of "old things floating around" that are keeping this issue open? We might be better off with a separate issue for those.
